devlib/platform/arm.py

```diff
@@ -13,9 +13,9 @@
 # limitations under the License.
```

Comment:
Changes in this file are nothing to do with a Gem5Platform, and their purpose is not clear. Is intended to be part of the commit?